tools/cloney

```diff
@@ -32,9 +32,11 @@ srcdir=$1
 destdir=$2
 
 PATH=/usr/bin
+CLONEY_MODE="symlink"
 
```

Comment:
Shouldn't this be
CLONEY_MODE?="symlink"
?

Reply:
Yes, I apparently killed that line out of my commit. Added